react/react1/week1/my-todo/src/component/toDo.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+
+  export const TodoList = [
+    {
+      description: 'Get out of bed',
+      deadline: 'Wed Sep 13 2017',
+    },
+    {
+      description: 'Brush teeth',
+      deadline: 'Thu Sep 14 2017',
+    },
+    {
+      description: 'Eat breakfast',
+      deadline: 'Fri Sep 15 2017',
+    },
+  ];
+
+
+export function TodoItem (props) {
+  return (
+    <div>
+    {props.TodoList.map((todo)=>{
+ return(
```

Comment:
This probably should be it's own component called todoItem or something like that

Reply:
thanks for the review Lucas